README.md

```diff
@@ -96,13 +95,12 @@ COMMANDS
   deploy   Create a new version of your documentation from the given file or URL.
   diff     Get a comparison diff with your documentation from the given file or URL.
   help     Display help for bump.
-  overlay  Apply an OpenAPI specified overlay to your API definition.
   preview  Create a documentation preview from the given file or URL.
 ```
 
  You can also get some help anytime by adding `--help` to any command. Example: `bump deploy --help`.
 
-### Prepare your Bump.sh account
+## Prepare your Bump.sh account
```

Comment:
this paragraph is part of the `## Usage` h2, it should stay as an h3 (or be moved before the `## Usage` paragraph I guess)